packages/bot/src/components/dialog.jsx

```diff
@@ -25,26 +26,40 @@ const Dialog = ({
                 <div className='bot-dialog__text'>
                     {children}
                 </div>
-                <Button
-                    className={classNames(
-                        'btn--primary',
-                        'bot-dialog__ok-button'
-                    )}
-                    text={'Ok'}
-                    onClick={closeModal}
-                    has_effect
-                />
-                {/* Todo Add array to send more buttons if requierd */}
+                <div className='bot-dialog__footer'>
+                    {onCancelButtonClick &&
+                        <Button
+                            className={classNames(
+                                'btn--secondary',
+                                'bot-dialog__cancel'
+                            )}
+                            text={'Cancel'}
+                            onClick={onCancelButtonClick}
+                            has_effect
+                        />}
+                    {onOkButtonClick &&
+                        <Button
+                            className={classNames(
+                                'btn--primary',
+                                'bot-dialog__ok'
+                            )}
+                            text={'Ok'}
```

Comment:
Please use the `translate` function (don't forget to import from `i18n.js`)

```suggestion
                            text={translate('Ok')}
```